.ci/.jenkins_python_full.yml

```diff
@@ -0,0 +1,3 @@
+PYTHON_VERSION:
```

Comment:
should this file contain the content of the `.ci/.jenkins_python.yml` file?

Reply:
right :) fixed in a3d1158